src/ExpressionEvaluator/Core/Source/ResultProvider/Expansion/MemberExpansion.cs

```diff
@@ -38,9 +38,9 @@ internal sealed class MemberExpansion : Expansion
             }
 
             var runtimeType = type.GetLmrType();
-            // Primitives, enums and null values with a declared type that is an interface have no visible members.
+            // Primitives, enums, function pointers, and null values with a declared type that is an interface have no visible members.
             Debug.Assert(!runtimeType.IsInterface || value.IsNull);
-            if (formatter.IsPredefinedType(runtimeType) || runtimeType.IsEnum || runtimeType.IsInterface)
+            if (formatter.IsPredefinedType(runtimeType) || runtimeType.IsEnum || runtimeType.IsInterface || runtimeType.IsFunctionPointer())
```

Comment:
I'm guessing there's no way to test this since C# can't represent function pointers.  Does reflection have a representation we could use?